### PR TITLE
Freedom Toy iteration 1

### LINKED ISSUE #40: Freedom Toys logo (Iteration 1)
Image

Image

Image

Image

Image

Image

______________

This issue size:
- Estimate SP: 2
- Actual SP: ?

### PR DESCRIPTION
**Pull request by naila135**
Resolve #40

### CODE REVIEW

File: PublisherBrand/Iterations/Iteration1.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+This is first iteration of creating logo for Freedom Toys made by designer according to [Product Owner view](https://github.com/sahlet-official/sleeplog/blob/develop/PublisherBrand/ProdOwnerView.md)<br>
+<empty line>
+Design was made acording to next lsit of requirements:
+- Product Owner wishes the logo to contain red elements;<br>
+- font of the logo must be simple;<br>
+- the logo must have a symbolic meaning.<br>
```
Comment (sahlet-official) [inline PublisherBrand/Iterations/Iteration1.md:6]: change dot "." to ";"
please

File: PublisherBrand/Iterations/Iteration1.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Several design variations were made. I devided them to 3 groups:
+- designs which were proposed by me;<br>
+- designs which were proposed by Product Owner;<br>
+- experimental designs.<br>
```
Comment (sahlet-official) [inline PublisherBrand/Iterations/Iteration1.md:11]: change dot "." to ";"
please

File: PublisherBrand/Iterations/Iteration1.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+This is first iteration of creating logo for Freedom Toys made by designer according to [Product Owner view](https://github.com/sahlet-official/sleeplog/blob/develop/PublisherBrand/ProdOwnerView.md)<br>
+<empty line>
+Design was made acording to next lsit of requirements:
+- Product Owner wishes the logo to contain red elements;<br>
+- font of the logo must be simple;<br>
+- the logo must have a symbolic meaning.<br>
+
+Several design variations were made. I devided them to 3 groups:
+- designs which were proposed by me;<br>
```
Comment (sahlet-official) [inline PublisherBrand/Iterations/Iteration1.md:9]: change "me" to "Designer"
please

File: PublisherBrand/Iterations/Iteration1.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+- designs which were proposed by Product Owner;<br>
+- experimental designs.<br>
+ 
+Design was made using Figma and can be seen [here](https://www.figma.com/file/ciA7yOiSqzYXabkYPZtjjy/FreedomToys?node-id=1%3A16&t=oA6l1WjzrlSPPJYq-1)
```
Comment (sahlet-official) [inline PublisherBrand/Iterations/Iteration1.md:13]: Put links to all sheets. cant see other sheets

**Review by sahlet-official — CHANGES_REQUESTED** [review]

File: PublisherBrand/Iterations/Iteration1.md (hunk 3 above)
Comment (naila135) [reply PublisherBrand/Iterations/Iteration1.md]: Yep, changed

File: PublisherBrand/Iterations/Iteration1.md (hunk 1 above)
Comment (naila135) [reply PublisherBrand/Iterations/Iteration1.md]: Done

**Review by sahlet-official — APPROVED** [review]

**Pull request merged**